pkg/cli/create/build.go

```diff
@@ -0,0 +1,223 @@
+package create
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	buildv1 "github.com/openshift/api/build/v1"
+	buildv1client "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
+	"github.com/openshift/library-go/pkg/image/reference"
+	"github.com/openshift/oc/pkg/helpers/env"
+)
+
+const BuildRecommendedName = "build"
+
+var (
+	buildLong = templates.LongDesc(`
+		Create a new build
```

Comment:
Why build and not buildconfig? You can than create build from BC, but we don't have that option in the other direction?

Reply:
Builds are like Jobs - useful on their own.  I might do a build config later but wanted to use build first to get some experience with the args (build config is a superset of build).

Reply:
That's fair, there's `oc create job test-job --from=cronjob/a-cronjob`